experimenter/experimenter/templates/changelog/overview.html

```diff
@@ -5,13 +5,17 @@
 {% endblock %}
 
 {% block content %}
-    <h1>Changes Overview</h1>
+  <div class="p-6">
+    <h1 class="text-3xl font-bold">Changes Overview</h1>
     <p>Experiment Slug: {{ experiment.slug }}</p>
-    {% for changelog in experiment.get_changelogs %}
-    <div>
-        <h2>{{ changelog.changed_on }}</h2>
-        <p><i class="fa-regular fa-pen-to-square"></i> : {{ changelog.message }}</p>  
-        <p><i class="fa-regular fa-user"></i> : {{ changelog.changed_by }}</p>
+    <div class="py-2 space-y-2">
+      {% for changelog in experiment.get_changelogs %}
+      <div class="bg-gray-200 rounded-md p-2">
+          <h2 class="font-semibold">{{ changelog.changed_on }}</h2>
+          <p><i class="fa-regular fa-pen-to-square"></i> : {{ changelog.message }}</p>  
+          <p><i class="fa-regular fa-user"></i> : {{ changelog.changed_by }}</p>
```

Comment:
I know fontawesome recommends using `<i>` for icons, but it's not great practice for semantic HTML. We should use `<span>`s here.